documentation/book/ref-maintenance-time-windows.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// assembly-maintenance-time-windows.adoc
+
+[id='ref-maintenance-time-windows-{context}']
+= Maintenance time windows configurations
+
+The maintenance time windows can be configured using the `Kafka.spec.maintenanceTimeWindows` property.
```

Comment:
Change lines 8 & 9 to:

"Maintenance time windows can be configured by entering an array of strings containing one or more {CronExpressions} in the `Kafka.spec.maintenanceTimeWindows` property."